index.html

```diff
@@ -365,14 +661,38 @@ <h4>
           short noindices>
         </ndef-record>
       </ndef-record>
+    </div>
+    </section>
+
+    <section> <h5>Signature records</h5>
+    <p>
+      <dfn>NDEF Signature</dfn> is defined [[NDEF-SIGNATURE]].
+      Its <a>TYPE field</a> contains "`Sig`" (`0x53`, `0x69`, `0x67`) and its
+      <a>PAYLOAD field</a> contains version, signature and a certificate chain.
+      Web NFC does not support this at the moment.
```

Comment:
Well we can read it as toArrayBuffer no?

Reply:
I don't know if we read it at all, or should we?

Reply:
I think we cannot go for full support now, and we'd need to spec it properly. Out of scope for this release.

Reply:
I would say that we should read all and allow to get bytes.. that offers a way for devs to work around us not having specced everything

Reply:
This would be a new feature for the current spec and needs to be discussed in its own issue. It's definitely not in the scope for this PR. I can remove all the sig record related text if that makes it better.

Reply:
No it's fine, but please file an issue

Reply:
OK, I will. I just thought this is not properly spec'd (just basic information) and maybe would be better to skip it now and add it later. But let's leave this text in for now and we'll work on it anyway soon.

Reply:
#363, #313

Reply:
Added a comment and reference to the issue.